mu_repo/__init__.py

```diff
@@ -69,6 +69,28 @@ def CreateConfig(config_file='.mu_repo'):
 def CreateParams(args, config_file='.mu_repo'):
     return Params(CreateConfig(config_file), args, config_file)
 
+
+def SearchConfigFile(start_dir, recurse_limit=20):
+    """Search upwards in the file system starting in start_dir for a ".mu_repo" file, until the maximum limit or
+    the root of the file-system is reached.
+    """
+    start_dir = os.path.normpath(os.path.abspath(start_dir))
+    while True:
```

Comment:
I think it'd read better if you did while recurse_limit > 0 and did a return None outside of the while if it didn't return.